### PR TITLE
CPDP-1625: ur banner with config and js

### PR DESCRIPTION
**Pull request by nubz**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -249,4 +249,9 @@ mixed-use {
   enabled = true
 }
 
+user-recruitment-banner {
+  enabled = true
```
Comment (AndyHWChung) [inline conf/application.conf:253]: do we need to set this to false in prod for now or are we ok with enabling it in prod straight away?

Comment (nubz) [reply conf/application.conf:253]: as i understand it there is a push to get this live asap

Comment (nubz) [reply conf/application.conf:253]: just confirmed with Ashleigh - the sooner the better and the link is good to go now

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**